news/664.feature

```diff
@@ -0,0 +1 @@
+force_add flag added to OmegaConf.update(), ensuring a the path is created even if it will result in insertion of new values into struct nodes.
```

Comment:
```suggestion
force_add flag added to OmegaConf.update(), ensuring that the path is created even if it will result in insertion of new values into struct nodes.
```